apps/mosaico/src/Article/Article.purs

```diff
@@ -64,6 +65,7 @@ type Props =
   , user :: Maybe User
   , mostReadArticles :: Array ArticleStub
   , latestArticles :: Array ArticleStub
+  , currentUrl :: String
```

Comment:
We actually do have the `shareUrl :: Maybe String` in Lettera's `Article` and `ArticleStub` types, however I noticed now they're not exposed in the purescript types. Could we use that instead?

Speaking of which, I'm not sure why it's a `Maybe` in Lettera, as we always know it. I need to look into it.

Reply:
Also manually mapping types from haskell to purescript is a pain 😅

Reply:
At least for eg. notFoundArticle in affresco, the shareUrl is quite naturally Nothing. Probably something similar in Lettera's end.

I'll update the code, it should end up being a lot nicer with that change.